app/src/org/commcare/android/util/DetailCalloutListener.java

```diff
@@ -13,4 +15,6 @@
     public void addressRequested(String address);
     
     public void playVideo(String videoRef);
+
+    public void performCallout(CalloutData callout, int id);
```

Comment:
Method documentation would be great, especially noting what "id" is, and what activity is expected to launch the callout